Libs/DICOM/Widgets/ctkDICOMBrowser.cpp

```diff
@@ -324,9 +331,11 @@ void ctkDICOMBrowserPrivate::init()
   this->ToolBar->setToolButtonStyle(Qt::ToolButtonTextUnderIcon);
 
   // Initialize Q/R widget
+#if QT_VERSION >= QT_VERSION_CHECK(5,0,0)
```

Comment:
The check should be done based on enabled/disabled features. We should avoid features disappearing silently based on detected Qt version.